includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -919,7 +919,7 @@ function() use ( $payment_fields ) {
 	 */
 	public function get_payment_method_ids_enabled_at_checkout( $order_id = null, $force_currency_check = false ) {
 		$capture                    = empty( $this->get_option( 'manual_capture' ) ) || $this->get_option( 'manual_capture' ) === 'no';
-		$capturable_payment_methods = $capture ? $this->get_upe_enabled_payment_method_ids() : [ 'card' ];
+		$capturable_payment_methods = $capture ? $this->get_upe_enabled_payment_method_ids() : [];
```

Comment:
Changing to an empty array seems not appropriate. We may filter and return `card` if `$this->get_upe_enabled_payment_method_ids()` array has `card` value.

Reply:
Yes, that was not correct.

I introduced a new function `WC_Payment_Gateway_WCPay->get_default_upe_enabled_payment_method_ids()` with the logic suggested above (in_array), so that it can be reused in the three places where it's needed.